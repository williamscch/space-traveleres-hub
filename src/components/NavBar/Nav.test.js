import { render } from '@testing-library/react';
import { BrowserRouter as Router } from 'react-router-dom';
import Nav from './Nav';

describe('Navbar', () => {
  test('Renders Nav component', () => {
    const tree = render(
      <Router>
        <Nav />
      </Router>,
    );
    expect(tree).toMatchSnapshot();
  });
});
