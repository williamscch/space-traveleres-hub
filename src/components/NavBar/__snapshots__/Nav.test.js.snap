// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar Renders Nav component 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <nav>
        <div
          class="logo-container"
        >
          <img
            alt="logo"
            class="nav-logo"
            src="planetColors.png"
          />
          <h1
            class="nav-h1"
          >
            Space Travelers Hub
          </h1>
        </div>
        <ul
          class="nav-ul"
        >
          <li
            class="nav-li"
          >
            <a
              aria-current="page"
              class="nav-link activated"
              href="/"
            >
              Rockets
            </a>
          </li>
          <span>
            |
          </span>
          <li
            class="nav-li"
          >
            <a
              class="nav-link"
              href="/missions"
            >
              Missions
            </a>
          </li>
          <span>
            |
          </span>
          <li
            class="nav-li"
          >
            <a
              class="nav-link"
              href="/my-profile"
            >
              My profile
            </a>
          </li>
        </ul>
      </nav>
    </div>
  </body>,
  "container": <div>
    <nav>
      <div
        class="logo-container"
      >
        <img
          alt="logo"
          class="nav-logo"
          src="planetColors.png"
        />
        <h1
          class="nav-h1"
        >
          Space Travelers Hub
        </h1>
      </div>
      <ul
        class="nav-ul"
      >
        <li
          class="nav-li"
        >
          <a
            aria-current="page"
            class="nav-link activated"
            href="/"
          >
            Rockets
          </a>
        </li>
        <span>
          |
        </span>
        <li
          class="nav-li"
        >
          <a
            class="nav-link"
            href="/missions"
          >
            Missions
          </a>
        </li>
        <span>
          |
        </span>
        <li
          class="nav-li"
        >
          <a
            class="nav-link"
            href="/my-profile"
          >
            My profile
          </a>
        </li>
      </ul>
    </nav>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
